src/main.js

```diff
@@ -179,7 +179,7 @@ class MiniGraphCard extends LitElement {
         ?labels-secondary=${config.show.labels_secondary === 'hover'}
         ?gradient=${config.color_thresholds.length > 0}
         ?hover=${config.tap_action.action !== 'none'}
-        style="font-size: ${config.font_size}px;"
+        style="font-size:${config.font_size}px;"
```

Comment:
Why all these changes in `main.js`? Makes code less readable

Reply:
Most of the changes I made were eliminating white spaces between CSS properties. Terser, even when configured aggressively, don't remove them, so I've noticed that and did it manually. Since they are just simple CSS rules, it shouldn't impact readability too much.

Reply:
Okay, Let's keep it like this then 👍